tests/integration/components/frost-text-test.js

```diff
@@ -16,29 +19,301 @@ describeComponent(
     })
 
     it('renders', function () {
-      this.render(hbs`{{frost-text}}`)
-      expect(this.$()).to.have.length(1)
+      this.render(hbs`
+          {{frost-text}}
+      `)
+
+      expect(
+        this.$('input').attr('tabindex'),
+        'tabindex is set'
+      ).to.eql('0')
+
+      expect(
+        this.$('input').attr('type'),
+        'type is set'
+      ).to.eql('text')
+
+      expect(
+        this.$('input').hasClass('frost-text-input'),
+        'class "frost-text-input" is set'
+      ).to.be.true
+
+      expect(
+        this.$('input').hasClass('left'),
+        'class "left" is set'
+      ).to.be.true
+
+      expect(
+        this.$('.frost-text-clear'),
+        'class "frost-text-clear" is set'
+      ).to.have.length(1)
+    })
+
+    it('sets align property', function () {
+      const align = 'right'
+
+      this.set('align', align)
+
+      this.render(hbs`
+          {{frost-text
+            align=align
+          }}
+      `)
+
+      expect(
+        this.$('input').hasClass(align),
+        'class "right" is set'
+      ).to.be.true
+    })
+
+    it('sets autocapitalize property', function () {
+      this.render(hbs`
+          {{frost-text
+            autocapitalize='on'
+          }}
+      `)
+
+      expect(
+        this.$('input').attr('autocapitalize'),
```

Comment:
same comment about changing the `.attr()` to `.prop()`. Please make that change for every occurrence.